### PR TITLE
Update peerDependency stylelint to ^9.0.0

### PR DESCRIPTION
**Pull request by brneto**
This PR contains a:

- [ ] **bugfix**
- [x] new **feature**
- [ ] **code refactor**
- [x] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

The motivation is the issue #149

### CODE REVIEW

**Comment by shellscape** [comment]
@brneto please sign the CLA to move forward

**Comment by brneto** [comment]
@jsf-clabot I have signed the CLA, but it still says I'm pending. What I've done wrong?

**Comment by shellscape** [comment]
@brneto probably signed it using a different account and/or email address. we see that often.

**Comment by brneto** [comment]
@shellscape How I fix that?

**Comment by brneto** [comment]
Make a new commit with other email address?

**Comment by brneto** [comment]
@jsf-clabot I have two e-mails to my GitHub account. Do I have to sign one CLA for each e-mail?

**Comment by shellscape** [comment]
(Please use the edit button on replies rather than consecutive messages. These aren't SMS messages :smile:)

No idea I'm afraid, I've never had the issue, and folks who were affected by it seemed to work it out, but never posted the solutions. Not being facetious when I say thing, but it's probably worth giving a google a try.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "stylelint-webpack-plugin",
-  "version": "0.10.5",
+  "version": "0.10.6",
```
Comment (shellscape) [inline package.json]: please don't modify the package version. project maintainers - those with publish ability - will take care of that. pretty much the standard on webpack-contrib projects (and from what I've gathered, most projects)

File: package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     "webpack-plugin"
   ],
   "peerDependencies": {
-    "stylelint": "^8.0.0",
+    "stylelint": "^8.0.0 || ^9.2.0",
```
Comment (shellscape) [inline package.json]: (I see this was from a previous commit - but still worth mentioning) It's curious that this dependency exists within `peerDependencies` and `dependencies` both. That makes the peerDep moot. @JaKXz  can you weight in as to what the intention should be here: do we require the user install it, or do we maintain the dependency?

Comment (JaKXz) [reply package.json]: The intent is that the user installs it themselves, as documented in the README - the peer dependency is to indicate what version of style lint is officially supported but really, any version that does not change the promise API is fine.

Comment (shellscape) [reply package.json]: Cool. So we need to remove that from `dependencies` and move it to `devDependencies` for use in testing.

Comment (brneto) [reply package.json]: So I need to remove it from peerDependencies and keep it only into devDependencies?

**Comment by shellscape** [comment]
@brneto ah looks like you deleted the file in a commit. that's not what was being asked for, just removing the file from the PRs changes. the lockfile shouldn't be modified in any way by a PR

**Comment by brneto** [comment]
I've put it back from commit `b244ff` (`chore(release): 0.10.5`). Sorry man for my mistake!

File: package.json (hunk 2 above)
Comment (shellscape) [reply package.json]: my apologies - it was the diff that was throwing me off, and made it look like stylelint was both in peerDeps and deps. we're all good on this comment thread

**Comment by abogaart** [comment]
Any chance this can be merged?

**Comment by shellscape** [comment]
@abogaart done. please don't ask about release, as this will happen when maintainers with those permissions are able to.

**Comment by abogaart** [comment]
Thanks for merging! I will keep quiet now :-)

**Pull request merged**